client/src/main/java/io/pravega/client/stream/impl/PendingEvent.java

```diff
@@ -24,7 +24,11 @@
  */
 @Data
 public class PendingEvent {
-    public static final int MAX_WRITE_SIZE = Serializer.MAX_EVENT_SIZE;
+    /**
+     * The max write size should differ to max event payload size, because the wire protocol adds extra bytes to the
```

Comment:
While this explanation is correct, it really doesn't describe what this field is.